modules/ROOT/pages/java-support.adoc

```diff
@@ -0,0 +1,160 @@
+
+* If you are a MuleSoft customer:
+
+. Update your connector that is generated from:
+  * xref:customer-connector-upgrade.adoc#upgrade-your-mule-sdk-connectors[Mule SDK]
+  * xref:customer-connector-upgrade.adoc#upgrade-your-rest-connect-connectors[REST Connect]
+. Test your connector using xref:customer-connector-upgrade.adoc#test-your-custom-connector-with-munit[MUnit].
+. xref:mule-runtime::deploying.adoc[Deploy your applications] to a target running on Mule 4.6 or later.
+
+[[policies]]
+== Policies
+
+The MuleSoft-included Mule Gateway policies are compatible with Java 17 beginning with the Mule runtime 4.6 release. These policies continue to be compatible with Java 8 until February 2025, so it's best to start updating your policies as soon as possible. 
+
+[IMPORTANT]
+To ensure your API proxies or Mule apps are protected when upgrading, upgrade your policies *before* upgrading your API proxies or Mule apps.
+
+For details about how to upgrade your policies, see xref:upgrade-policies-proxies.adoc#upgrading-automated-policies[Upgrading Automated Policies] and xref:upgrade-policies-proxies.adoc#upgrading-api-level-policies[Upgrading API-Level Policies].
+
+[[api-proxies]]
+== API Proxies
+
+The MuleSoft-included API proxies are compatible with Java 17 beginning with the Mule runtime 4.6 release. These API proxies continue to be compatible with Java 8 until February 2025, so it’s best to start updating your API proxies as soon as possible. 
+
+The steps to upgrade are a little different, depending on which deployment model you use. 
+
+* If you use Basic endpoint, deploy the adapted application to the server from Mule runtime and connect it to API Manager using autodiscovery. For more information, see <<mule-runtime>>.
+* If you use a Basic endpoint API instance to update your instance, update the Mule application connecting to your API instance.
+
+[IMPORTANT]
+To ensure your API proxies or Mule apps are protected when upgrading, upgrade your API policies *before* upgrading your API proxies or Mule apps.
+
+For details about how to upgrade your API proxies, see xref:upgrade-policies-proxies.adoc#upgrading-api-proxies[Upgrading API Proxies].
+
+[[mule-maven-plugin]]
+== Mule Maven Plugin 
+
+If you are using Mule Maven Plugin (MMP) to deploy your apps, configure the deployment to use Java 17.  
+
+When deploying to CloudHub, MMP deploys the latest build version of a release train when it's given a major and minor version. MMP has a new Java version property to explicitly deploy to a specified Java version.
+
+When deploying to Runtime Fabric (RTF) and CloudHub 2.0, MMP accepts the entire tag of the build so you can use the correct semantic version (SemVer) in your deployment. 
+
+See the following documentation for more information:
+
+* xref:cloudhub-2/ch2-deploy-maven.adoc[Deploying Apps to CloudHub 2.0 Using the Maven Plugin]
+* xref:runtime-fabric/deploy-maven-4.x.adoc[Deploying Apps to Runtime Fabric Using the Maven Plugin]
```

Comment:
are you linking to this? https://docs.mulesoft.com/mule-runtime/latest/deploy-to-rtf

if so:

xref:mule-runtime::deploy-to-rtf.adoc[Deploying Applications to Runtime Fabric Using the Mule Maven Plugin]